rules/precompiled_apple_resource_bundle.bzl

```diff
@@ -110,7 +110,7 @@ def _precompiled_apple_resource_bundle_impl(ctx):
         input_plists = ctx.files.infoplists,
         output_pkginfo = None,
         output_plist = output_plist,
-        output_discriminator = None,
+        output_discriminator = "precomp",
```

Comment:
Would "bundle" be more obvious here?

Reply:
Good point, will update!

Reply:
Updated both instead https://github.com/bazel-ios/rules_ios/pull/743/commits/3ff7b868ab8c7af39e94762318b95a1c7bb2f021